src/Sylius/Behat/Page/Admin/Crud/EditPage.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Crud;
+
+use Behat\Mink\Session;
+use Sylius\Behat\Page\SymfonyPage;
+use Symfony\Component\Routing\RouterInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class EditPage extends SymfonyPage implements EditPageInterface
+{
+    /**
+     * @var string
+     */
+    private $resourceName;
```

Comment:
Making getter for one property does not bock anything, I don't understand why should we wait.